### ISSUE TITLE
[DocDB] Speedup remote bootstrap by moving checksum to background

### ISSUE THREAD

**Issue by bmatican** [yugabyte/yugabyte-db#12352]
### Description

```
src/yb/tserver/remote_bootstrap_service.cc
...
void RemoteBootstrapServiceImpl::FetchData
...
  session->rate_limiter().UpdateDataSizeAndMaybeSleep(info.data.size());
  uint32_t crc32 = Crc32c(info.data.data(), info.data.length());
```

We should add some timing info, even if just for debugging, to understand the time spent on fetching data vs writing it to disk vs computing the checksum. It's possible that for low network latency scenarios, the checksuming being in the critical path is slowing down the RBS process.

For high latency scenarios, the RTT might be wasting more cycles, so there we might need some form of streaming / pipelining.

**Comment by bmatican** [comment]
@rthallamko3 @basavaraj29 note, I think @karan-yb  added some timing info, perhaps after this issue was filed. We should review those, from an actual cluster, to see if it’s worth even doing this work, ie does it actually add a considerable overhead or not

**Comment by rthallamko3** [comment]
Yep, We just changed assignment to @basavaraj29 to add it to his backlog.
